src/middlewared/middlewared/scripts/vendor_service.py

```diff
@@ -0,0 +1,75 @@
+import hashlib
+import os
+import subprocess
+import sys
+
+from truenas_api_client import Client
+
+
+def load_envvars(envvars_file: str):
```

Comment:
We already have `dotenv` available in scale, so you can do something like this
```
>>> from dotenv import load_dotenv
>>> load_dotenv('/root/.env')
True
>>>
```

Reply:
Also if the transient service should have this env, then you can also specify it to systemd to load environment from specific file

Reply:
Unfortunately I couldn't find a `systemd-run` option equivalent to setting the `EnvironmentFile` property in a service file, and I don't think `--property=EnvironmentFile=` works. I'll look into `dotenv`.